tox.ini

```diff
@@ -26,7 +26,7 @@ deps =
     # AttributeError: 'module' object has no attribute 'PEP257Checker'
 
 commands =
-    flake8 --max-complexity 10 sacred
+    flake8 --ignore=W504 --max-complexity 10 sacred
```

Comment:
I agree to ignoring this, but I suggest to move it to [`setup.cfg`](https://github.com/IDSIA/sacred/blob/master/setup.cfg#L10)